configure.ac

```diff
@@ -1028,6 +1038,9 @@ echo "Event handlers:"
 AM_COND_IF([ENABLE_SAMPLEEVH],
 	[echo "    Sample event handler:  yes"],
 	[echo "    Sample event handler:  no"])
+AM_COND_IF([ENABLE_GELFEVH],
+	[echo "    Gelf event handler:    yes"],
+	[echo "    Gelf event handler:    no"])
```

Comment:
Just a nit, but this should appear at the end of the list. The order is not alphabetic, but depending on when they were added.

Reply:
As a side note, isn't it called GELF (all-caps)?